### ISSUE TITLE
Black Screen Death after third try

### ISSUE THREAD

**Issue by NoNameProvided** [PFCKrutonium/Windows-10-Login-Background-Changer#15]
Ok, so I tried this. I have set a picture and it worked, then set an another one and it was still working. But after setting the third one when I tried to watch it by locking my computer I got a black screen with the cursor only visible. After restart the problem still existed, and general solutions for "blackscreen with cursor on Windows 8.1 and 10" on the internet haven't helped so I had to reinstall my system.

**Comment by Krutonium** [comment]
Next time you have this issue, assuming it happens again, boot up your recovery media, get to a command line, and go to C:\Windows\SystemResources\Windows.UI.Logon. In there, delete the Windows.UI.Logon.pri and replace it with a copy of Windows.UI.Logon.pri.bak

We will be adding pri verification soon.

**Comment by Toyz** [comment]
Which verison of the program the newest one or the old one the new one is a differnt UI, post a picture of which one your using or just the name ex "W10 BG Logon Changer"

This can tell me more information and attempt to fix this bug so it doesn't happen to other users

**Comment by arieljannai** [comment]
Hi,

Do you know in which version it occurs? It's kind of scary!
I've tried it once, and don't want to take the risk :fearful: 

I have version 1.0.2.0 according to the title bar.
Risky or a good version?

Thanks :smile: 
Ariel

**Comment by Toyz** [comment]
That's the good version

**Comment by Sojiro84** [comment]
Ok, so I downloaded the latest version, extracted it and ran it. Got a nice wallpaper, applied it and now I get a nice flickering login screen. Computer is broken.

The fix mentioned here should help me get my PC to a usuable state? I just go into advanced recovery and open a command prompt?

**Comment by darth62969** [comment]
yes, the latest update changed a few things with the logon ui, best option is to use the regesty hack mentioned in #58 or uninstall and prevent the update from installing. i am working on figuring out how to disable this update in 10 home.

**Comment by Sojiro84** [comment]
I just saw same other fixes as well. I can't just edit the registry right? I first need to load the hive before I can edit it in the advanced recovery?

The regedit and restoring the .bak both seem pretty easy. Found the commands to restore the .bak and I need to google to find the command to load the registry hive.

Man, I was afraid I had to reinstall my windows. Luckily there is a fix. Pfff, why do I always want to customize the OS! :P

I finally had a windows installed for 2 months. World record for me haha.

Oh well, will fix it later tonight. TIA.
